spec/features/merchants/items/index_spec.rb

```diff
@@ -14,8 +14,8 @@
   end
 
   it 'the item name is a link to the merchant items show page' do
-    click_link("#{@item1.name}")
+    click_link("#{@item.name}")
```

Comment:
This looks like it fixes an issue I had come across this morning. Thank you for fixing it!

Reply:
It popped up when I ran rspec and I thought it was a mistake I had made earlier, then when I thought about it later I wasn't sure if I accidentally changed something one of y'all wrote  😅 
Either way I'm glad its good now!